### DISCUSSION TITLE
Suggestion: Add visual regression tests with screenshots

### DISCUSSION THREAD

**Discussion by gauthierdc** [mobile-dev-inc/Maestro#1002] [Ideas]
It would be great if we could compare some screenshots betweens uploads to visualy check for regression like in [storybook](https://storybook.js.org/docs/react/writing-tests/visual-testing).

**Comment by axelniklasson** [comment]
Hi @gauthierdc, thanks for the suggestion! This is on our roadmap and there is a feature request here https://github.com/mobile-dev-inc/maestro/issues/762 where you can follow along. Thanks!
